packages/sdk/src/evm/core/classes/marketplace-auction.ts

```diff
@@ -459,6 +459,32 @@ export class MarketplaceAuction {
     };
   }
 
+  /**
+   * Get the buffer in basis points between offers
+   */
+  public async getBidBufferBps(): Promise<BigNumber> {
+    return this.contractWrapper.readContract.bidBufferBps();
+  }
+
+  /**
+   * returns the minimum bid a user can place to outbid the previous highest bid
+   * @param listingId - the listing id of the auction
+   */
+  public async getMinimumNextBid(
+    listingId: BigNumberish,
+  ): Promise<BigNumberish> {
+    const currentBidBuffer = await this.getBidBufferBps();
+    const winningBid = await this.getWinningBid(listingId);
+    const listing: AuctionListing = await this.getListing(listingId);
+    if (winningBid) {
+      const winningBidPrice = winningBid.currencyValue.value;
+      return winningBidPrice.add(currentBidBuffer.mul(winningBidPrice));
+    } else {
+      const price = listing.reservePrice;
+      return price.add(currentBidBuffer.mul(price));
+    }
+  }
```

Comment:
hey @jnebab thx for doing this work, take a look at this suggestion to optimize the async calls a bit more

```suggestion
  public async getMinimumNextBid(
    listingId: BigNumberish,
  ): Promise<BigNumberish> {
    // we can fetch both of these at the same time using promise.all
    const [currentBidBuffer, winningBid] = await Promise.all([
      this.getBidBufferBps(),
      this.getWinningBid(listingId),
    ]);

    if (winningBid) {
      const winningBidPrice = winningBid.currencyValue.value;
      return winningBidPrice.add(currentBidBuffer.mul(winningBidPrice));
    } else {
      // we can delay getting the listing until we know there is no winning bid (since we don't need it for the calculation unless there is no winning bid)
      const listing = await this.getListing(listingId);
      const price = listing.reservePrice;
      return price.add(currentBidBuffer.mul(price));
    }
  }
```